src/orion/algo/dehb/dehb.py

```diff
@@ -0,0 +1,509 @@
+"""
+:mod:`orion.algo.dehb.dehb -- DEHB
+==================================
+
+Module for the wrapper around DEHB: https://github.com/automl/DEHB.
+"""
+
+
+import logging
+from collections import OrderedDict, defaultdict
+from copy import deepcopy
+from typing import List, Optional
+
+import numpy as np
+
+from orion.algo.dehb.brackets import SHBracketManager
+from orion.algo.dehb.logger import remove_loguru
+
+remove_loguru()
```

Comment:
```suggestion
```
(since it's a no-op anyway)

Reply:
Removed in https://github.com/Epistimio/orion/commit/ebfff61c98c062202355f2ce3c308661e9fc80a1